counsel.el

```diff
@@ -2707,7 +2755,14 @@ INITIAL-DIRECTORY, if non-nil, is used as the root directory for search.
 EXTRA-RG-ARGS string, if non-nil, is appended to `counsel-rg-base-command'.
 RG-PROMPT, if non-nil, is passed as `ivy-read' prompt argument."
   (interactive)
-  (let ((counsel-ag-base-command counsel-rg-base-command))
+  (let ((counsel-ag-base-command counsel-rg-base-command)
+        (counsel--grep-tool-look-around
+         (pcase counsel-rg-pcre-flag
+           ('auto (when (version<= "0.10.0" (counsel--rg-version))
```

Comment:
Nitpick: by convention, `when` is for control-flow and `and` is for boolean expressions.